src/node_contextify.cc

```diff
@@ -1647,6 +1647,12 @@ static const auto throws_only_in_cjs_error_messages =
         "await is only valid in async functions and "
         "the top level bodies of modules"};
 
+static const auto maybe_top_level_await_errors =
+    std::array<std::string_view, 2>{// example: `func(await 1);`
+                                    "missing ) after argument list",
+                                    // example: `if(await 1)`
+                                    "SyntaxError: Unexpected"};
```

Comment:
Did the linter/formatter format the code this way?

Reply:
@jasnell 
Could you take another look when you have a moment?